### PR TITLE
Clarified phone number

### PR DESCRIPTION
**Pull request by cloveras**

### CODE REVIEW

File: docs/swagger.yaml [hunk 1]
```diff
@@ -583,7 +583,17 @@ components:
...
-          description: Target customer phone number. 8 digits.
+          description: |-
+            The customer's phone number, if available.
+            Format: 8 digits.
+            **Please note:** The phone number is optional, and should only be sent if it is
+            already known.
+            Users should never be asked for the phone number, as they will
+            either be automatically app-switched to the Vipps app, or they will be presented
+            with the landing page.
+            If the phone numnber is needed: Use `scope` and the Userinfo API:
```
Comment (cloveras) [inline docs/swagger.yaml:594]:
```suggestion
            If the customer's phone number is needed by the merchant: Use `scope` and the Userinfo API:
```

**Comment by rebekaburnett** [comment]
Be sure to use "Squash and merge" to make a single commit with the final change.

File: docs/swagger.yaml [hunk 2]
```diff
@@ -583,7 +583,17 @@ components:
       properties:
         customerPhoneNumber:
           type: string
-          description: Target customer phone number. 8 digits.
+          description: |-
+            The customer's phone number, if available.
+            Format: 8 digits.
+            **Please note:** The phone number is optional, and should only be sent if it is
```
Comment (rebekaburnett) [inline docs/swagger.yaml:589]:
```suggestion
            **Please note:** The phone number is optional and should only be sent if it is
```

File: vipps-ecom-api.md [hunk 3]
```diff
@@ -365,9 +365,19 @@ An express payment example with more parameters provided:
 }
 ```
 
-**Please note:** Do not send sensitive information in the `transactionText` field.
-See
-[Datatilsynet's information](https://www.datatilsynet.no/rettigheter-og-plikter/personopplysninger/)
+**Please note:** The phone number is optional, and should only be sent if it is
```
Comment (rebekaburnett) [inline vipps-ecom-api.md:368]:
```suggestion
**Please note:** The phone number is optional and should only be sent if it is
```

**Review by rebekaburnett — APPROVED** [review]
Thanks!! This is great! A couple comments.

**Pull request merged**